pkg/apis/openebs.io/v1alpha1/cstor_volume_replica.go

```diff
@@ -52,6 +52,8 @@ type CStorVolumeReplica struct {
 type CStorVolumeReplicaSpec struct {
 	TargetIP string `json:"targetIP"`
 	Capacity string `json:"capacity"`
+	// ZvolWorkers represents number of threads that executes client IOs
```

Comment:
Can we mention how can one determine the best value given an infrastructure?
Does it default to any value?

Reply:
There is default values , but for the tunnings we are going to have a doc/blog with details how someone can determine these values based on there cluster cc @vishnuitta